include/CL/sycl/device.hpp

```diff
@@ -208,9 +208,11 @@ class device
       \todo
   */
   template <info::device Param>
-  auto get_info() const {
+  size_t get_info() const {
     // Forward to the version where the info parameter is not a template
     //return get_info<typename info::param_traits_t<info::device, Param>>(Param);
+	assert(false); //non implemented
```

Comment:
static_assert(false, "...") raises an error at compile blocking the compilation process.